src/Symfony/Component/ExpressionLanguage/Node/BinaryNode.php

```diff
@@ -147,6 +147,10 @@ public function evaluate($functions, $values)
             case '*':
                 return $left * $right;
             case '/':
+                if (0 == $right) {
+                    throw new \DivisionByZeroError('Division by zero');
+                }
+
                 return $left / $right;
             case '%':
```

Comment:
this one needs the same check, isn't it?

Reply:
You are right. It's necessary for PHP5.
Done